no-computer.md

```diff
@@ -2,54 +2,122 @@
 
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+This section describes what the non-computer DSL is and who uses it.
 
+Abacuses are a DSL for calculating numbers and storying numbers. It is a DSL for
+people that do not have calculators, for people that want to develop their
+arithmetic abilities, and was the main calculating tool for millions of people
+for thousands of years (until electronic calculators became a thing).
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
-
+_What need is met by this DSL? Who does it help? What is that person's
+experience like before using this DSL? What would their experience be like after
+they started using this DSL?_
+
+The needs were obvious in the before-calculators age. It has a simple system
+that can represent large numbers, and it speeds up things like accounting work.
+The experiences of people before abacus was probably tedious, having to write or
+memorize numbers on their way to a calculation (and it's not like they did not
+calculate large things back then!). After they created the abacus, people can
+do those calculations much easier and faster.
+
+The current use of abacus is largely educational. Many kindergarten and primary
+school age children in places like China and Japan learn the abacus to practice
+and improve their arithmetic abilities. So the need here is an abstraction to
+learning and practicing arithmetic, which reinforces the arithmetic its users
+learn in use. The abacus does this fairly well as it is entertaining to little
+children and is an effective tool to reinforce their arithmetic skills.
```

Comment:
This is cool. It's pretty interesting how a tool that's been around for thousands of years is still in use today, and has evolved to fill a different role (educational as opposed to practical).